@xen-orchestra/backups/writers/_AbstractReplicationWriter.js

```diff
@@ -0,0 +1,4 @@
+exports.AbstractReplicationWriter = (BaseClass = Object) =>
+  class AbstractReplicationWriter extends BaseClass {
```

Comment:
Add a constructor which will receive all params passed to replication writers.

You can also store them all this value on the instance.